README.md

```
@@ -19,158 +17,81 @@ BDD, API testing and CI/CD integration.
 | AssertJ | Fluent assertions |
 | WebDriverManager | Automatic driver management |
 | Maven | Build and dependency management |
-| CodeRabbit | Automated AI code reviews on pull requests |
+| GitHub Actions | Continuous integration |
+| CodeRabbit | Automated AI code reviews |
 | JUnit 5 | Unit test execution |
+| SLF4J + Log4j2 | Structured logging |
 
 ## Project Structure
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)